rotkehlchen/chain/ethereum/names.py

```diff
@@ -3,11 +3,63 @@
 from eth_utils import to_checksum_address
 
 from rotkehlchen.chain.ethereum.decoding.constants import ETHADDRESS_TO_KNOWN_NAME
+from rotkehlchen.chain.ethereum.manager import EthereumManager
+from rotkehlchen.constants import ENS_UPDATE_INTERVAL
 from rotkehlchen.db.addressbook import DBAddressbook
 from rotkehlchen.db.dbhandler import DBHandler
 from rotkehlchen.db.ens import DBEns
+from rotkehlchen.errors.misc import BlockchainQueryError, RemoteError
 from rotkehlchen.globaldb import GlobalDBHandler
-from rotkehlchen.types import AddressbookType, ChecksumEvmAddress, EnsMapping, SupportedBlockchain
+from rotkehlchen.types import (
+    AddressbookType,
+    ChecksumEvmAddress,
+    EnsMapping,
+    SupportedBlockchain,
+    Timestamp,
+)
+from rotkehlchen.utils.misc import ts_now
+
+
+def find_ens_mappings(
```

Comment:
you just moved this here, right? Any other changes?

Reply:
No, no changes